test/collections/VectorListTest.ooc

```diff
@@ -225,10 +225,19 @@ VectorListTest: class extends Fixture {
 			list free()
 		})
 		this add("VectorList sort", func {
-			//TODO Current way of sorting not supported by Rock
+			list := VectorList<Int> new()
+			list add(8); list add(16); list add(32)
```

Comment:
This isn't consistent with the above tests, but please do `list add(8) . add(16) . add(32)` instead.